docs/pages/deployment/pex.rst

```diff
@@ -82,6 +82,19 @@ E.g., in the example above, a claim named ``fullName`` is added to the introspec
 The following is an example OAuth2 Token Introspection response containing the ``fullName`` claim from the Presentation Definition
 (some fields are omitted for brevity):
 
+.. code-block:: json
+
+  {
+    "iss": "did:web:example.com",
+    "active": true,
+    "scope": "example_scope",
+    "fullName": "John Doe"
+  }
+
+If you want introspection to return part of a string, you can use the ``pattern`` regular expression filter in the field definition with a capture group.
+Token introspection will return the value of the first capture group in the regular expression, instead of the full match, e.g.;
+``{"role": "Admin level 4"}`` with the following pattern filter: ``"pattern": "Admin level ([0-9])"`` will return ``"role": "4"`` (given the field ID ``role``).
```

Comment:
This is not enough information for me to understand what to do. Adding an example of a (partial) constraints field would help.